src/fandango/evolution/algorithm.py

```diff
@@ -247,6 +247,9 @@ def _perform_crossover(
                 crossovers = self.crossover_operator.crossover(
                     self.grammar, parent1, parent2
                 )
+                if crossovers is None:
+                    return None
+
```

Comment:
This then just forwards how crossover doesn't return unchanged elements anymore.